src/Theme/Space.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
+import { DisplayBox, DisplayBoxWrapper } from '../DisplayBox';
+import { DisplayBoxTitle, DisplayBoxSubTitle, DisplayBoxBody } from '../DisplayBox/DisplayBox.styles.ts';
+
+export const spaces = [
+  { size: '4px', name: 'xxsmall' },
+  { size: '8px', name: 'xsmall' },
+  { size: '16px', name: 'small' },
+  { size: '24px', name: 'base' },
+  { size: '32px', name: 'largel' },
+  { size: '48px', name: 'xlarge' },
+  { size: '64px', name: 'xxlargel' },
+];
+
+<Meta title="Theme|Space" />
+
+# Spacing
+
+This is the spacing that will be used throught all edozo apps (customer facing). The design files can be found [here](https://www.figma.com/file/OpCqZzeuBbb8NBshcsBRpc/Edozo-maps-DS?node-id=155%3A1235).
+
+<Story name="Default">
+  <DisplayBoxWrapper>
+    {spaces.map(space => (
+      <DisplayBox size={space.size} radius="xsmall">
```

Comment:
Can you change radius to something more like borderRadius so it is less confusing